pkg/sessionctx/vardef/tidb_vars.go

```diff
@@ -1344,6 +1347,8 @@ const (
 	DefTiFlashMaxBytesBeforeExternalSort    = -1
 	DefTiFlashMemQuotaQueryPerNode          = 0
 	DefTiFlashQuerySpillRatio               = 0.7
+	DefTiFlashHashJoinVersion               = joinversion.TiFlashHashJoinVersionDefVal
+	DefTiFlashUseHashJoinV2                 = joinversion.TiFlashUseHashJoinV2DefVal
```

Comment:
It seems a little confuse to have 2 variables for TiFlashUseHashJoinV2, there is some risk that the 2 variables is not consistent. I think we can delete this bool variable?

Reply:
Done.